cg/services/analysis_starter/configurator/file_creators/nextflow/params_file/models.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+
+from cg.models.nf_analysis import WorkflowParameters
+
+
+class RarediseaseParameters(WorkflowParameters):
+    """Model for Raredisease parameters."""
+
+    target_bed_file: str
+    analysis_type: str
+    save_mapped_as_cram: bool
+    vcfanno_extra_resources: str
```

Comment:
Implement change from #4330: Remove `skip_germlinecnvcaller` as it will be fetched from servers